components/Templates/page_layout.html

```diff
@@ -0,0 +1,140 @@
+  <link rel="stylesheet" href="#">
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+</head>
+
+<body>
+
+  <div class="wrap">
+    <div class="background-header"></div>
+    <aside>
+      <ul>
+        <li> 1 </li>
+        <li> 2 </li>
+        <li> 3 </li>
+        <li> 4 </li>
+        <li> 5 </li>
+        <li> 6 </li>
+        <li> 7 </li>
+      </ul>
+    </aside>
+    <div class="container">
+      <div class="content">
+        <h2>Lorem ipsum!</h2>
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis
+          aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
+      </div>
+      <!--/.content-->
+    </div>
+    <!--/.container-->
+  </div>
+  <!--/.wrap-->
+  <footer class="main-footer">
+    <span>placeholder for footer</span>
+  </footer>
+</body>
+<style>
+  /* =================================
+  Base Styles
+==================================== */
+
+  * {
+    box-sizing: border-box;
+  }
+
+  body {
+    font-family: sans-serif;
+    line-height: 1.6;
+    color: #3a3a3a;
+  }
+
+  p {
+    font-size: .95em;
+    margin-bottom: 1.5em;
+  }
+
+  h2,
+  h3,
+  a {
+    color: #093a58;
+  }
+
+  h2,
+  h3 {
+    margin-top: 0;
+  }
+
+  /* =================================
+  Base Layout Styles
+==================================== */
+
+  .container {
+    padding-left: 1em;
+    padding-right: 15em;
+  }
+
+  .background-header {
+    background-image: url(../../img/placeimg_968_190_nature.jpg);
+    background-repeat: no-repeat;
+    background-size: cover;
+    padding-top: 10.99em;
+    padding-bottom: 1.5em;
+    margin-bottom: 40px;
+  }
+
+  .main-footer {
+    text-align: center;
+    padding: 2em 0;
+    background: #d9e4ea;
+  }
+
+  /* placeholder for the nav */
+  aside {
+    width: 25.5%;
+    padding-left: .5rem;
+    margin-left: 2.5rem;
+    float: right;
+    box-shadow: inset 5px 0 5px -5px #29627e;
+    color: #29627e;
+  }
+
+  /* =================================
+  Media Queries
+==================================== */
+  @media (max-width: 500px) {
+    .container {
+      min-width: 700px;
```

Comment:
In my testing of this, I found that this line caused the page to look "squished" on smaller screens. I'd consider removing this, though let me know if I'm missing something!

Reply:
Thank you! I added this media query so the page look the same as http://www.casamarianella.org/about-us/ on small screen.